Improve Config Reference docs (#42)
DefinedNet/nebula-docs: docs/config/logging.mdx

  #disable_timestamp: true
  #timestamp_format: "2006-01-02T15:04:05.000Z07:00"
```

## logging.level

<Pill className="mb-24">Default: info</Pill>

options are `panic`, `fatal`, `error`, `warning`, `info`, or `debug`.

## logging.format

<Pill className="mb-24">Default: text</Pill>

options are `json` or `text`

## logging.disable_timestamp

<Pill className="mb-24">Default: False</Pill>

Disable timestamp logging. Useful when output is redirected to logging system that already adds timestamps.

## logging.timestamp_format

```yml

> # timestamp format is specified in Go time format, see:

Author: sounds good to me 👍

> # timestamp format is specified in Go time format, see:

Author: Pushed a diff that implements this in some fashion 👍

  #disable_timestamp: true
  #timestamp_format: "2006-01-02T15:04:05.000Z07:00"
```

## logging.level

<Pill className="mb-24">Default: info</Pill>

> options are `panic`, `fatal`, `error`, `warning`, `info`, or `debug`.

Reviewer: ```suggestion
Controls the verbosity of logs.  The options are `panic`, `fatal`, `error`, `warning`, `info`, or `debug`.
```

## logging.format

<Pill className="mb-24">Default: text</Pill>

> options are `json` or `text`

Reviewer: ```suggestion
Controls the logging format.  The options are `json` or `text`.
```

## logging.disable_timestamp

<Pill className="mb-24">Default: False</Pill>

> Disable timestamp logging. Useful when output is redirected to logging system that already adds timestamps.

Reviewer: ```suggestion
Disables timestamp logging. Useful when output is redirected to a logging system that already adds timestamps.
```

## logging.timestamp_format

`timestamp_format` is specified in Go time format, see:

> https://golang.org/pkg/time/#pkg-constants

Reviewer: ```suggestion
https://golang.org/pkg/time/#pkg-constants.
```

The default when `format: json` is set is an [RFC3339 timestamp](https://datatracker.ietf.org/doc/html/rfc3339), e.g. `"2006-01-02T15:04:05Z07:00"`.

> The default when `format: text` is either seconds since beginning of execution when a TTY is attached, otherwise: `"2006-01-02T15:04:05Z07:00"` (RFC3339).

Reviewer: ```suggestion
When `format: text` is set, if a TTY is attached the default is seconds since beginning of execution, otherwise it is RFC3339 (e.g. `"2006-01-02T15:04:05Z07:00"`).
```
